flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/rest/message/statement/FetchResultsRequestBody.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.gateway.rest.message.statement;
+
+import org.apache.flink.runtime.rest.messages.RequestBody;
+
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonInclude;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.Nullable;
+
+/** {@link RequestBody} for fetching job result. */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class FetchResultsRequestBody implements RequestBody {
+
+    private static final String FIELD_NAME_MAX_FETCH_SIZE = "max_fetch_size";
```

Comment:
Please make sure that _all_ request/response body fields use camelCase.

Reply:
I agree that it would be nice to use consistent naming conventions for the rest field in 2.0, and we should start to do that early. Is there already a consensus that camelCase will be used in long term? Could you point us to it?

Reply:
See the discussion in https://issues.apache.org/jira/browse/FLINK-15787

I also made sure all recently added REST APIs followed camelCase.

Reply:
@zentol, thanks a lot for the pointer.

@WencongLiu, let's also add a hotfix commit and fix this for the APIs we added in FLINK-28161.

Reply:
@zentol @xintongsong thanks, I'll fix it.